tmt-app/src/Components/Home.js

```diff
@@ -1,16 +1,24 @@
-import React from "react";
+import React, { useState, lazy, Suspense } from "react";
 import "../Styling/Home.css";
 import { PrimaryBtn, FlexContainer } from "../Styling/CustomStyling.js";
-import { Link, Container, Typography } from "@mui/material";
+import { Link, Container, Typography, Button } from "@mui/material";
 import MilkTeaBlob from "../Images/MilkTeaBlob.png";
 import LycheeRectangle from "../Images/LycheeRectangle.png";
 import MilkTeaSwiggle2 from "../Images/MilkTeaSwiggle2.png";
 import ThaiTeaSwiggle1 from "../Images/ThaiTeaSwiggle1.png";
-import ArrowRightAltIcon from "@mui/icons-material/ArrowRightAlt";
+import ArrowDownwardIcon from "@mui/icons-material/ArrowDownward";
+import ArrowUpwardIcon from "@mui/icons-material/ArrowUpward";
 import Breadcrumb from "./Breadcrumb";
 import { Link as RouterLink } from "react-router-dom";
+const MajorCareerList = lazy(() => import("./MajorCareerList.js"));
+const GuideSidebar = lazy(() => import("./GuideSidebar.js"));
 
 function Home() {
+  const [displayGuide, setDisplayGuide] = useState(false);
+  const changeDisplay = () => {
+    setDisplayGuide(!displayGuide);
```

Comment:
setDisplayGuide((previous) => !previous)

Best practice to make changes that are dependent on the previous value from within the callback.